### DISCUSSION TITLE
是否能修改暗码?/Can I modify the 5776733 to other number?

### DISCUSSION THREAD

**Discussion by joyol** [LSPosed/LSPosed#2374] [Q&A]
是否能修改暗码?/Can I modify the 5776733 to other number?

**Comment by hui-shao** [comment]
目前在 APP 内似乎并没有提供设置选项，如果要修改，可以通过修改源码，自行打包 APK 的方式进行。

相应代码在以下文件的第289行。
https://github.com/LSPosed/LSPosed/blob/master/daemon/src/main/java/org/lsposed/lspd/service/LSPosedService.java

P.S. 5776733 恰好对应 9 键输入的 LSPOSED 单词，个人感觉挺好记的。
